govcd/edgegateway_lb_service_monitor.go

```diff
@@ -0,0 +1,187 @@
+package govcd
```

Comment:
edgegateway_lb_service_monitor.go - such naming practise seems new in our packet.

Reply:
We already have the "parent" `edgegateway.go`. Please elaborate and suggest alternatives.

Reply:
I can make `lb_service_monitor.go` but to me the `LB` is part of edgegateway so this gives a better hint.

Reply:
Actually, that's pretty nice suggestion, but I now realize that @vbauzysvmware must have meant the usage of underscores, as currently we collapse the words into names like `edgegateway.go`, `catalogitem.go`, `externalnetwork.go`, etc.

Reply:
Is it `lbservicermonitor.go` then?

Reply:
Looks consistent to me, @vbauzysvmware ?

Reply:
yep, I can live with that :) and yet how we call load balancer? `lb.go? or loadbalancer.go - just thinking loud, how all names will endup - so we will see better if we don't miss :)

Reply:
Actually there is no such thing as purely load balancer (`lb.go`). Actual load balancer is created from blocks (lbvirtualserver is where it ties it all together):
* `lbservicermonitor.go`
* `lbserverpool.go`
* `lbappprofile.go` (or `lbapplicationprofile.go`)
* `lbvirtualserver.go`

Reply:
renamed

Reply:
P.s. lbapplicationprofile.go is better